cpp/src/neighbors/ball_cover.cu

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2021-2025, NVIDIA CORPORATION.
```

Comment:
These are new files and I just realized we have 2021 in the date here. Mind changing these to just 2025?